server/api/notification.php

```diff
@@ -65,17 +65,17 @@
          $userID = $bodyData['userID'] ?? "";
```

Comment:
@mxs4321 Any reason we shouldn't use PHP 7 because once this merges it will be required.

Reply:
@lzganec Would `?:` work the same here?

Reply:
@BLamy it would need additional null check if you were to use ?: instead of ??. Since at times I had several of these it was much faster doing the null coalescing than ternary with isset(). If for some reason there is a problem with anyone on php7 I guess we will have to switch, but we have requested a server that has php7, have we not?